assets/src/scss/layout/_navbar.scss

```diff
@@ -117,9 +117,8 @@ $navbar-default-height: 60px;
   }
 
   @include small-and-less {
-    &.open {
-      width: 100%;
-    }
+    width: 100vw;
```

Comment:
I have a small glitch if you take `width` out of the `.open` selector, if you resize your window to a smaller width the left menu appears and disappears for no reason
![Peek 2022-01-26 14-18](https://user-images.githubusercontent.com/617346/151170331-87d48f4c-d8de-4e00-b46e-90d61f64ca07.gif)

Reply:
I can reproduce this too. It happens at exactly `767px` width.

Reply:
Oh sorry! I completely missed this conversation. Working on this